### PR TITLE
[DRAFT] primary set up

### PR DESCRIPTION
**Pull request by fereshtehnm**
Added configuration for npm package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "aibrary",
+  "version": "1.0.0",
```
Comment (AlirezaMorsali) [inline package.json]: let's start with 0.0.1

File: package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "description": "Custom Node.js wrapper for OpenAI APIs",
```
Comment (AlirezaMorsali) [inline package.json]: This should be:
The official TypeScript library for the AiBrary API

File: src/index.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  baseURL?: string;
+}
+
+export class AIBrary {
+  private openai: OpenAI;
+
+  constructor({ apiKey, baseURL }: AILibraryConfig) {
+    this.openai = new OpenAI({
+      apiKey,
+      baseURL: baseURL || "https://api.openai.com/v1",
```
Comment (AlirezaMorsali) [inline src/index.ts]: The base url is our url: api.aibrary...

**Review by AlirezaMorsali — COMMENTED** [review]
I left some comments.
I recommend looking at our python package for inspiration and some details, e.g. getting models from the api and ...

File: package.json (hunk 2 above)
Comment (fereshtehnm) [reply package.json]: done

File: package.json (hunk 1 above)
Comment (fereshtehnm) [reply package.json]: done

File: src/index.ts (hunk 3 above)
Comment (fereshtehnm) [reply src/index.ts]: also done
However, I will add it to .env file later

**Pull request closed without merging**